string labels
int32 px
int32 py
geometry_msgs/Point bbox1
geometry_msgs/Point bbox2



